android/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,8 @@
     <application
 
         android:label="Clima"
-        android:icon="@mipmap/ic_launcher">
+        android:icon="@mipmap/ic_launcher"
+        android:roundIcon="@mipmap/ic_launcher_round">
```

Comment:
From https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive:
>  You must only use the android:roundIcon attribute if you require a different icon asset for circular masks, if for example the branding of your logo relies on a circular shape.

This means that `android:roundIcon` should only be used if the icon were to be different between the normal icon and the round icon. This does not apply to us, since there is no difference between both icons, so this should be removed, along with all the round icon assets.